backend/tracim_backend/lib/utils/image_process.py

```diff
@@ -0,0 +1,57 @@
+from fractions import Fraction
+import typing
+
+from PIL import Image
+from PIL import ImageOps
+
+
+class ImageRatio:
+    def __init__(self, width: int, height: int):
+        self.width = width
+        self.height = height
+
+    def ratio(self) -> Fraction:
+        return Fraction(self.width, self.height)
+
+    def sized_to_ratio(self, image_size: "ImageSize") -> "ImageSize":
+        width = image_size.width
+        height = image_size.height
+        if image_size.ratio() == self.ratio():
+            pass
+        elif image_size.ratio() > self.ratio():
+            # INFO - G.M - 2021-01-21 - we do not allow new width to be
+            # bigger than previous one,
+            # in that specific edge case, we do not render proper ratio.
```

Comment:
Do you have an example where this happens? (same for the else)

Reply:
not really cannot reproduce error i do get with height, just done same check as height because i'm paranoid.